provider/README.md

```diff
@@ -358,7 +361,10 @@ The `/status_changes` API should allow querying status changes with the followin
 | --------------- | ------ | --------------- |
 | `event_time` | `YYYY-MM-DDTHH`, an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) extended datetime representing an UTC hour between 00 and 23. | All status changes with an event time occurring within the hour. For example, requesting `event_time=2019-10-01T07` returns all status changes where `2019-10-01T07:00:00 <= status_change.event_time < 2019-10-01T08:00:00` UTC. |
 
-If the data does not exist or the hour has not completed, `/status_changes` shall return a `404 Not Found` error.
+For hours that have not yet completed or during which the provider was not operating
+in a municipality `/status_changes` shall return a `404 Not Found` error.
+`/status_changes` shall return an empty array of status change events for
```

Comment:
`/status_changes` must return a Payload with an an empty array of status change events
`{ "status_changes": [] }` for

Reply:
Yes